Python/PRP/Region/pyVolumeIsect.h

```diff
@@ -18,8 +18,10 @@
 #define _PYVOLUMEISECT_H
 
 #include "PyPlasma.h"
+#include <PRP/Region/plVolumeIsect.h>
 
 PY_WRAP_PLASMA(VolumeIsect, class plVolumeIsect);
+PY_WRAP_PLASMA(SinglePlane, class plConvexIsect::SinglePlane);
 PY_WRAP_PLASMA(ConvexIsect, class plConvexIsect);
```

Comment:
Since you have now included the declarations of `plVolumeIsect`, `plConvexIsect::SinglePlane`, and `plConvexIsect`, you may safely delete their forward declarations. In other words, remove the `class` from each of these lines.